charts/orchestrator-infra/Chart.yaml

```diff
@@ -13,4 +13,4 @@ maintainers:
 type: application
 sources:
   - https://github.com/redhat-developer/rhdh-chart
-version: 0.0.1
+version: 0.0.2
```

Comment:
Is this version bump needed? The changes here only affect the `backstage` chart to me.

Reply:
Yes, chart linting failed on it